src/native-common/FrontLayerViewManager.tsx

```diff
@@ -179,6 +179,17 @@ export class FrontLayerViewManager {
         );
     }
 
+    private _getOverlayContext(rootViewId?: string | null): PopupStackContext {
```

Comment:
The return type here should be PopupStackContext | undefined. That should allow you to remove the incorrect "as PopupStackContext" cast below.